File: src/main/java/server/ServerGame.java
package server;

import common.Game;
import talaria.common.Talaria;

public class ServerGame extends Game {

    public static ServerGame instance = null;

    public ServerWorld world;

    /**
     * Updates the world.
     */
    public ServerGame() {
        world = new ServerWorld("foo", 0, 0);
    }

    /**
     * Updates the world.
     */
    public void update() {
        world.update();
        // This will sync the clients to the server.
        Talaria.INSTANCE.getServer().sendEntityToAll(world);
    }
}

File: src/main/java/server/ServerGame.kt
package server

import common.Game
import talaria.common.Talaria

class ServerGame : Game() {

    var world: ServerWorld = ServerWorld("foo", 0, 0)

    /**
     * Updates the world.
     */
    override fun update() {
        world.update()
        // This will sync the clients to the server.
        Talaria.server?.sendEntityToAll(world)
    }

    companion object {

        @JvmStatic
        lateinit var instance: ServerGame
    }
}
